src/yang/ietf-bgp-policy.yang

```diff
@@ -21,7 +21,7 @@ module ietf-bgp-policy {
   organization
     "IETF IDR Working Group";
   contact
-    "WG Web:   <http://tools.ietf.org/wg/idr>
+    "WG Web:   <http://datatracker.ietf.org/wg/idr>
```

Comment:
Repeating comments from before, it's ideal to keep unrelated bits of diff from pull requests.  This probably should go under a separate issue to update the copyright dates.

Reply:
The tools are flagging the Copyright date as an issue, and they use the last modified date to decide whether the date should be updated.